atomics/include/desul/atomics/Generic.hpp

```diff
@@ -247,6 +247,16 @@ atomic_fetch_oper(const Oper& op,
     done_active = DESUL_IMPL_BALLOT_MASK(done);
   }
   return return_val;
+// FIXME_SYCL not implemented
+#elif defined(__SYCL_DEVICE_ONLY__)
+  (void) op;
+  (void) dest;
+  (void) scope;
+  (void) return_val;
+  (void) done;
+
+  assert(false);
+  return val;
```

Comment:
Could do that.  Since we know it isn't supported might be better to static_assert out if something tries to instantiate it if we can work out a way to do it.